src/DaySlot.jsx

```diff
@@ -150,11 +170,13 @@ let DaySlot = React.createClass({
         <div
           key={'evt_' + idx}
           style={{...xStyle, ...style}}
-          title={label + ': ' + title }
+          title={(typeof label ? label + ': ' : '') + title }
```

Comment:
Remember my change that let messages be React components.

In the rare event that messages.allDay has been customized with a Component as a message and one of these rare multi-day but not all-day events exists, then label may end up a React component instead of a string.
![screen shot 2016-02-16 at 12 16 20 pm](https://cloud.githubusercontent.com/assets/53399/13089784/2d4ffcb4-d4a7-11e5-8bf4-a9b4ac8dd60d.png)

It's a rare situation and the textual label doesn't seem that important so I just decided to drop it in that rare situation.